tactis/model/decoder.py

```diff
@@ -41,8 +35,17 @@ def _split_series_time_dims(x: torch.Tensor, target_shape: torch.Size) -> torch.
     return x.view(target_shape)
 
 
+def _simple_linear_projection(input_dim: int, output_dim: int) -> nn.Sequential:
+    layers = [nn.Linear(input_dim, output_dim)]
+    return nn.Sequential(*layers)
```

Comment:
The `Sequential` has no effect, except if you need to assume that `net[0].input_dim` is always a valid way to get that information.

Reply:
No, it has 3 effects:

1. Torch does not move lists automatically to the GPU. It only moves those lists wrapped up torch.nn.ModuleList or torch.nn.Sequential
2. It makes calling the layers easier as I do not have to iterate through them manually
3. It allows for easy automatic model parallelization in the future